angular/projects/spark-angular/src/lib/components/sprk-card/sprk-card.stories.ts

```diff
@@ -31,15 +22,6 @@ export default {
 const modules = {
   imports: [
     SprkCardModule,
-    SprkCardMediaModule,
```

Comment:
Only SprkCardModule needed now, see this PR description for more information on this.